README.md

```diff
@@ -50,7 +50,7 @@ export RUST_LOG=debug,substrate_api_client=warn,sp_io=warn,ws=warn,integritee_se
 ./integritee-service --clean-reset -r 2002 -p 9979 -w 2001 run 2>&1 | tee worker.log
 ```
 
-wait until you see the worker synching a few blocks. then check MRENCLAVE and update bot-community.py constants accordingly
+wait until you see the worker syncing a few blocks. then check MRENCLAVE and update bot-community.py constants accordingly
```

Comment:
Remove that line completely. We don't have a bot-community  (that was from the encounter worker)

Reply:
Actually, also the ipfs and node part are deprecated. I'd recommend removing these completely and only refer to the book, check out the current master README.md